### ISSUE TITLE
Parsing error: Identifier 'headerTag' has already been declared

### ISSUE THREAD

**Issue by guybroadhurst** [superhi/editor-issues#29]
**Describe the bug**
I'm getting a parsing error when I really shouldn't within my JS. The editor is giving this error "Parsing error: Identifier 'headerTag' has already been declared" when I am just defining 'headerTag' on the first line of code. I can get around this by redefining the const as 'headTag' for example. Also doesn't seem to like my const underneath, 'toggleHeader' but will work with 'toggleHead'. It is strange because I was working on a different project yesterday and it was fine when I had consts with the same identifiers. Even when I've commented all the rest of the code below out I still get the error.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to https://future-editor.superhi.com/projects/88653 within the header.js script
2. Identify the first const as 'headerTag'
3. will highlight as parsing error. 

**Expected behaviour**
not get a parsing error on the  

**Screenshots**
![Screenshot 2021-02-17 at 08 00 00](https://user-images.githubusercontent.com/69678024/108174150-00f19f00-70f7-11eb-8799-a6a43fa113f0.png)
![Screenshot 2021-02-17 at 08 00 20](https://user-images.githubusercontent.com/69678024/108174160-0353f900-70f7-11eb-990c-edff83006bf5.png)
![Screenshot 2021-02-17 at 08 01 26](https://user-images.githubusercontent.com/69678024/108174165-03ec8f80-70f7-11eb-862a-71be15e1de9a.png)
![Screenshot 2021-02-17 at 08 01 49](https://user-images.githubusercontent.com/69678024/108174168-04852600-70f7-11eb-81c3-5b3786b36a47.png)

**Device (please complete the following information):**
 - OS: macOS 10.15.7
 - Browser: Google Chrome Version 88.0.4324.150

**Additional context**

**Comment by artcommacode** [comment]
Hey @guybroadhurst did you rename the file or have a separate file with this JavaScript already in it? (I see you have a few empty JS files here) Do you still get these errors if you refresh the Editor?

**Comment by guybroadhurst** [comment]
Hi @artcommacode ah didn't even think to refresh the window. That does seem to have fixed it when I've changed them back to be headerTag etc. Hadn't renamed the file, and just set the other JS files up but haven't got to putting anything in them yet, but when I attached the other JS files to the HTML I did copy the previous tags and just renamed the JS in the src by selecting it on the drop-down with the ... that pops up. So don't know if it was trying to attach the same JS file three times possibly?

**Comment by guybroadhurst** [comment]
Hi @artcommacode, I think I've got a similar thing happening on another project I've just created. 

It's showing errors in a file I was working on in another project beforehand. I've just created this new project and it's all just as the default. Not created or linked any assets or files yet. 

This does go away when I refresh the browser. 

![Screenshot 2021-02-19 at 14 11 25](https://user-images.githubusercontent.com/69678024/108515267-cb96ae00-72bc-11eb-8858-133b8fb339b7.png)
